<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RWindow</class>
 <widget class="QMainWindow" name="RWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget_Note">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="RBaseTab">
       <attribute name="title">
        <string>New Page</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="RBaseTab2">
       <attribute name="title">
        <string>New Page2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSupport"/>
    <addaction name="separator"/>
    <addaction name="actionSend_Feedback"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Editor Theme</string>
     </property>
     <addaction name="actionLight"/>
     <addaction name="actionDark"/>
     <addaction name="actionMonokai"/>
    </widget>
    <addaction name="actionFont"/>
    <addaction name="menuTheme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About RNote</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>Help and Support</string>
   </property>
  </action>
  <action name="actionSend_Feedback">
   <property name="text">
    <string>Send Feedback</string>
   </property>
  </action>
  <action name="actionTestCode">
   <property name="text">
    <string>TestCode</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Theme Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Absolute Dark</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font ...</string>
   </property>
  </action>
  <action name="actionMonokai">
   <property name="text">
    <string>Monokai</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
